ci/run_test/run_test_post_always.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This is a post test processing script for post processing the
+# run_test.sh stage CI run
+
+set -ex
+
+# shellcheck disable=SC1091
+source ./.build_vars.sh
+
+DAOS_BASE=${SL_PREFIX%/install*}
+rm -rf $DAOS_BASE/run_test.sh $DAOS_BASE/vm_test
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]